Dockerfile.rhel7

```diff
@@ -0,0 +1,12 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+
+WORKDIR /go/src/github.com/openshift/cluster-resource-override-admission
+COPY . .
+RUN make build
```

Comment:
The [makefile](https://github.com/openshift/cluster-resource-override-admission/blob/master/Makefile) in this repo doesn't seem to include a `build` target, is this provided elsewhere?

Reply:
yes, it's provided by `library-go` build machinery
```
# Include the library makefile
include $(addprefix ./vendor/github.com/openshift/library-go/alpha-build-machinery/make/, \
	golang.mk \
	targets/openshift/images.mk \
)
```
does this answer your question?